krabs/krabs/wstring_convert.hpp

```diff
@@ -0,0 +1,31 @@
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
let's add the Microsoft copyright above this as well:

`// Copyright (c) Microsoft. All rights reserved.`